src/index.ejs

```diff
@@ -40,6 +40,7 @@
                       return a;
                     }, {});
       var useModal = options.useModal === "true"
+      var liveCapture = options.liveCapture === "true"
```

Comment:
is this needed?

Reply:
This is test code that I have added to facilitate acceptance. Will get rid of it.